armi/physics/neutronics/latticePhysics/latticePhysicsWriter.py

```diff
@@ -260,7 +260,11 @@ def _getAllNuclidesByCategory(self, component=None):
                 # Homogeneous number densities and temperatures
                 nucTemperatureInC = self._getAvgNuclideTemperatureInC(nucName)
 
-            density = max(dens, self.minimumNuclideDensity)
+            # density = max(dens, self.minimumNuclideDensity)
```

Comment:
Please remove the commented out code
https://terrapower.github.io/armi/developer/standards_and_practices.html#remove-commented-out-code

Reply:
reverted the commit for this change

Reply:
https://github.com/terrapower/armi/pull/1288/commits/1b40c6bd93a70896a578ff79329957766b7db34f